Makefile

```diff
@@ -1,5 +1,11 @@
 include build/config.mak
 
+ifeq (${DEVICE}, n0110)
```

Comment:
Should be MODEL instead of DEVICE.
The way it's done doesn't seem right to me. What if I want to dev an external app using the simulator (yup it's possible). Maybe you should add a `FORCE_EXTERNAL` option?

```
ifeq (${MODEL}, n0110)
  apps_list = ${EPSILON_APPS}
else
  apps_list = $(foreach i, ${EPSILON_APPS}, $(if $(filter external, $(i)),,$(i)))
endif

ifdef FORCE_EXTERNAL
  apps_list = ${EPSILON_APPS}
endif
```

Reply:
I don't know it External Apps is currently supported for the simulator. However, we can create a list of supported devices in order to make easy adding support of external apps for some models/simulator.

Reply:
The default app (called App) is located in `apps/external/app`. Having a FORCE_EXTERNAL option is better imo because it's for a specific case (as all compilations options should be). Not putting external on simulator by default isn't a problem, I think. We just have to make some doc to tell the people "if you want external in simulator or n0100 use apps/external/app".